src/styles/base/_column.scss

```diff
@@ -13,10 +12,12 @@
 	height: 100%;
 	width: 100%;
 	min-height: 768px;
+	display: flex;
+  align-items: center;
 	background-color: $tint-color;
 	position: absolute;
-	display: inline-block;
 	@media (min-width: 1224px) {
 		width: 50%;
+		display: inline-block;
```

Comment:
Do you want to `display flex` or `display inline-block`? 🤔 
Keep one declaration